### ISSUE TITLE
Command not found Exception, After a successful use. Windows 8 (64bit) 

### ISSUE THREAD

**Issue by Xalbi** [coreybutler/nvm-windows#84]
i just installed nvm, i encoutred the problem #68 and solved it by copy/pasting the "settings.txt"  
After that  when i wanted to check node version: 
![image](https://cloud.githubusercontent.com/assets/13153320/9681582/68d67594-52ff-11e5-8f1b-eb7812c7f274.png)

what am I supposed to do?
Thnx.

**Comment by utahkay** [comment]
I have the same experience on Windows 7 - nvm-windows apparently not updating my path.

**Comment by enrichit** [comment]
Did you use the node installer previously?

For me it was a case of removing the old `node` directory located at `C:\Program Files\node` manually. See issue #18 for more details on this, even if you uninstall the global node it doesn't delete `node_modules` from your program files.

Also you don't need the hack described in #68. All you have to do is close **ALL** cmd windows and reopen them and the system path should update will take effect.

**Comment by Xalbi** [comment]
I found a solution to this issue,
In fact, i forgot to delete the 'node' value from the PATH, before installing nvm,
maybe that what caused a conflict: Windows try to look at that value, before checking the new variable (%NVM_HOME%;%NVM_SYMLINK%) 

Any way,

After cleaning the PATH, I did uninstalled and Reinstalled nvm:
i kept the default settings, so the root is [C:\Users\****\AppData\Roaming\nvm] and the path is [C:\Program Files]
This time, i got something like 'C:\Program' not a command.
so an other time i (un&re)installed nvm with a path and root without space.
and it worked. :D
